projects/Mallard/src/components/article/html/css.ts

```diff
@@ -144,12 +145,43 @@ const makeCss = ({ colors, theme }: CssProps, contentType: string) => css`
       line-height: 1.4;
       margin-bottom: 15px;
     }
+    
     .app h2 {
-      font-size: ${px(getScaledFont('headline', 1).lineHeight)};
-      line-height: ${px(getScaledFont('headline', 1).lineHeight * 1.1)};
-      margin-bottom: ${px(metrics.vertical)};
-      margin-top: ${px(metrics.vertical * 2.5)};
+        font-size: ${contentType === 'immersive' ? '24px' : '20px'};
```

Comment:
Would recommend using a html data type to help target h2 with a specific template. It should already be at the `.app` level from my template PR.

e.g. `.app[data-type='${Article.immersive}']` h2 { }`

This means you can have the standard H2 styling and the immersive one defined separately which would be cleaner. There are some not so good examples in the header file.

Reply:
Nice suggestion, I've just updated this 👍